src/main/java/com/client/core/base/tools/entitychanger/impl/StandardEntityChanger.java

```diff
@@ -0,0 +1,147 @@
+package com.client.core.base.tools.entitychanger.impl;
```

Comment:
how did this diff get into your PR? This is not aligned with the latest version of my PR. Can you undo this diff?

Reply:
We discussed in the meeting that we are just going to leave this here. Then your changes will overwrite mine